### ISSUE TITLE
Fixing failing tests on master

### ISSUE THREAD

**Issue by justin808** [shakacode/react_on_rails#1429]
Tests are failing on master...

Here's some research on this this:

To repro:

```
bundle exec rake run_rspec:examples
cd gen-examples/examples/basic-server-rendering
rails s -e test
tail -f log/test.log
```

The problem is that the server-bundle.js is getting generated with chunks. The SSR only pulls in the top level file. It doesn't deal with chunks.

```json
{
  "entrypoints": {
    "hello-world-bundle": {
      "assets": {
        "js": [
          "/packs-test/js/runtime.js",
          "/packs-test/js/134.js",
          "/packs-test/js/hello-world-bundle.js"
        ],
        "css": [
          "/packs-test/css/hello-world-bundle.css"
        ]
      }
    },
    "server-bundle": {
      "assets": {
        "js": [
          "/packs-test/js/runtime.js",
          "/packs-test/js/134.js",
          "/packs-test/js/server-bundle.js"
        ],
        "css": [
          "/packs-test/css/server-bundle.css"
        ]
      }
    }
  },
  "hello-world-bundle.css": "/packs-test/css/hello-world-bundle.css",
  "hello-world-bundle.js": "/packs-test/js/hello-world-bundle.js",
  "js/134.js": "/packs-test/js/134.js",
  "js/134.js.LICENSE.txt": "/packs-test/js/134.js.LICENSE.txt",
  "runtime.js": "/packs-test/js/runtime.js",
  "server-bundle.css": "/packs-test/css/server-bundle.css",
  "server-bundle.js": "/packs-test/js/server-bundle.js"
}
```

**Comment by justin808** [comment]
The root cause was the updated bin/webpacker binstub causes a search for config/webpack/test.js which didn't exist.

Then the default webpack config was used which used code-splitting.

We really need a better SSR failure message when this happens!



```
Puma caught this error: wrong number of arguments (given 1, expected 0) (ActionView::Template::Error)
/Users/justin/shakacode/react-on-rails/react_on_rails_2/lib/react_on_rails/prerender_error.rb:12:in `initialize'
/Users/justin/shakacode/react-on-rails/react_on_rails_2/lib/react_on_rails/helper.rb:486:in `exception'
/Users/justin/shakacode/react-on-rails/react_on_rails_2/lib/react_on_rails/helper.rb:486:in `raise'
/Users/justin/shakacode/react-on-rails/react_on_rails_2/lib/react_on_rails/helper.rb:486:in `rescue in server_rendered_react_component'
/Users/justin/shakacode/react-on-rails/react_on_rails_2/lib/react_on_rails/helper.rb:482:in `server_rendered_react_component'
/Users/justin/shakacode/react-on-rails/react_on_rails_2/lib/react_on_rails/helper.rb:424:in `internal_react_component'
/Users/justin/shakacode/react-on-rails/react_on_rails_2/lib/react_on_rails/helper.rb:57:in `react_component'
/Users/justin/shakacode/react-on-rails/react_on_rails_2/gen-examples/examples/basic-server-rendering/app/views/hello_world/index.html.erb:2:in `_app_views_hello_world_index_html_erb__1506071207523435023_13680'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/base.rb:244:in `public_send'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/base.rb:244:in `_run'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/template.rb:157:in `block in render'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/activesupport-7.0.2/lib/active_support/notifications.rb:208:in `instrument'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/template.rb:361:in `instrument_render_template'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/template.rb:155:in `render'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/renderer/template_renderer.rb:65:in `block (2 levels) in render_template'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/activesupport-7.0.2/lib/active_support/notifications.rb:206:in `block in instrument'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/activesupport-7.0.2/lib/active_support/notifications/instrumenter.rb:24:in `instrument'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/activesupport-7.0.2/lib/active_support/notifications.rb:206:in `instrument'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/renderer/template_renderer.rb:60:in `block in render_template'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/renderer/template_renderer.rb:75:in `block in render_with_layout'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/activesupport-7.0.2/lib/active_support/notifications.rb:206:in `block in instrument'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/activesupport-7.0.2/lib/active_support/notifications/instrumenter.rb:24:in `instrument'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/activesupport-7.0.2/lib/active_support/notifications.rb:206:in `instrument'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/renderer/template_renderer.rb:74:in `render_with_layout'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/renderer/template_renderer.rb:59:in `render_template'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/renderer/template_renderer.rb:11:in `render'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/renderer/renderer.rb:61:in `render_template_to_object'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/renderer/renderer.rb:29:in `render_to_object'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/rendering.rb:117:in `block in _render_template'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/base.rb:270:in `in_rendering_context'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/rendering.rb:116:in `_render_template'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionpack-7.0.2/lib/action_controller/metal/streaming.rb:216:in `_render_template'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.2/lib/action_view/rendering.rb:103:in `render_to_body'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionpack-7.0.2/lib/action_controller/metal/rendering.rb:46:in `render_to_body'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionpack-7.0.2/lib/action_controller/metal/renderers.rb:142:in `render_to_body'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionpack-7.0.2/lib/abstract_controller/rendering.rb:25:in `render'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionpack-7.0.2/lib/action_controller/metal/rendering.rb:30:in `render'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionpack-7.0.2/lib/action_controller/metal/instrumentation.rb:22:in `block (2 levels) in render'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/3.0.0/benchmark.rb:308:in `realtime'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/activesupport-7.0.2/lib/active_support/core_ext/benchmark.rb:14:in `ms'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionpack-7.0.2/lib/action_controller/metal/instrumentation.rb:22:in `block in render'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionpack-7.0.2/lib/action_controller/metal/instrumentation.rb:91:in `cleanup_view_runtime'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionpack-7.0.2/lib/action_controller/metal/instrumentation.rb:21:in `render'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionpack-7.0.2/lib/action_controller/metal/implicit_render.rb:35:in `default_render'
/Users/justin/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/actionpack-7.0.2/lib/action_controller/metal/basic_implicit_render.rb:6:in `block in send_action'
<internal:kernel>:90:in `tap'
```

**Comment by justin808** [comment]
Fixed by #1431
